ArcadiaParties.API/Controllers/UsersController.cs

```diff
@@ -55,5 +61,39 @@ public async Task<IActionResult> GetCurrentUser(CancellationToken cancellationTo
 
             return Ok(user);
         }
+
+        [ProducesResponseType(typeof(UserFromAssistantDTO), StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [SwaggerOperation(
+             Summary = "Returns current authenticated user"
+        )]
+        [Authorize]
+        [HttpGet]
+        [Route("GetUserFromAssistant")]
+        public async Task<IActionResult> GetUserFromAssistant(CancellationToken cancellationToken)
```

Comment:
Why do you need this? Client shouldn't know where data is retrieved from. Client should call `GetCurrentUser` endpoint and user with all data from Assistant, ServiceDesk or wherever else should be returned.

Reply:
Renamed

Reply:
Renamed for what? There should be no specific methods to get data from Assistant or somewhere else. All this should be performed only internally, client application should know about single method - /GetCurrentUser which should encapsulate data from all needed sources.